tutorials/nginx-reverse-proxy-docker.md

```diff
@@ -0,0 +1,392 @@
+---
+title: Running an NGINX Reverse Proxy with Docker and Let's Encrypt on Google Compute Engine
+description: Learn to serve multiple websites simultaneously in a single Compute Engine instance with Docker and NGINX.
+author: tswast
+tags: Compute Engine, NGINX, Docker, Lets Encrypt
```

Comment:
the correct spelling would be `Let's Encrypt` unless there's a restriction in tags.